### PR TITLE
Detach db cli

### LINKED ISSUE #41: Empty ASSIGNMENTS_FOLDER variable doens't work
In case the environment variable is not set, the current directory for the relative path (line 6 in `db/models/custom/assignment.js`) is taken to be the path from which the script was invoked.

I.e. to make it work I had to go to db/assignments/dutch-auction and run node `../../../cli/cli.js`.

Is there a way to tell the `path` module that we want "current directory" to mean "the script's directory" instead of "the directory in which `node` was executed"?

### PR DESCRIPTION
**Pull request by cnasikas**
Fixes #41

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -157,6 +164,7 @@ Commands:
   cli.js generate <command>  Generate <aux>
```
Comment (dionyziz) [inline README.md:164]: Shouldn't this be `cli.js generate aux <assignment>`? Also on the right hand side "<aux>" should read "aux", it's not a placeholder.

File: README.md [hunk 2]
```diff
@@ -157,6 +164,7 @@ Commands:
...
   cli.js get <command>       Get an entity <assignment>
```
Comment (dionyziz) [inline README.md:165]: This should state `cli.js get <id>`. The description should say "Get assignment given by <id>". Generally, the convention in UNIX help is that things that are within "<>" are variables holding user input and that descriptions only reference things that are part of the command.

File: cli/cmds/stats.js [hunk 3]
```diff
@@ -31,6 +32,21 @@ const getRanking = async (argv, order = 'ASC') => {
   return table.toString()
 }
 
+const getScore = async (argv) => {
+  const { ParameterizedAssignment } = require('blockchain-course-db').models
+  const totalSolved = await ParameterizedAssignment.count(
+    {
+      where: { studentId: argv.id, solved: true }
+    })
```
Comment (dionyziz) [inline cli/cmds/stats.js]: nit: `)` in its own line, like below.

**Review by dionyziz — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (cnasikas) [reply README.md:164]: This is the help message printed automatically by the cli framework that we use. `cli.js generate <command> Generate <aux>` means that after `generate` you should put a command and the available commands are only `aux`

Comment (dionyziz) [reply README.md:164]: I understand that this is the help message.

This is not the UNIX convention. In the UNIX convention, the things in `<>` designate something to be typed by the user, which is named into a variable; in this case `<command>` is the placeholder and `command` is the variable. The description of what it does, which is "Generate <aux>" contains a *different* variable `<aux>`, which is unclear what it means here. You should either say that `<command> can only be "aux"` or just replace `<command>` with `aux` and have the following help `cli.js generate aux <assignment>        Generate aux for assignment.`  as I suggested.

See any standard manpage to verify the style -- e.g. `man wget`.

Comment (cnasikas) [reply README.md:164]: I am not in control of how the help message will be printed. I can only change the description ie `Generate <aux>`, which I will. Try `node cli.js --help` and `node cli.js generate --help` to see the difference.

Comment (dionyziz) [reply README.md:164]: I understand that the messages are printed by the library, that's not what I am asking. I'm suggesting that you change the help message text.

Comment (cnasikas) [reply README.md:164]: Ok!

**Comment by cnasikas** [comment]
Depends on #42

**Pull request merged**